Branch-SDK/Branch-SDK/Branch.h

```diff
@@ -87,6 +87,7 @@ typedef enum {
                                                            andAlias:(NSString *)alias;
 
 + (void)setDebug;
+- (void)simulateInitFinished;
```

Comment:
I'm sure this is only used for testing, but I feel like we shouldn't expose this method. Is there another way we could trigger this without exposing it?